src/components/Input/type.tsx

```diff
@@ -0,0 +1,31 @@
+export enum InputType {
```

Comment:
The name of the enum must be `INPUT_TYPE`

Reply:
I`ve changed INPUT_TYPE in root and in export in plases, where this enum emplemented